instance/management/commands/reprovision_buckets.py

```diff
@@ -53,6 +54,13 @@ def handle(self, *args, **options):
             LOG.info('Reprovisioning %s S3 bucket to update bucket and iam configuration', ref.instance.name)
 
             # Reprovisioning will update the necessary configuration
-            ref.instance.provision_s3()
-
-            LOG.info('Reprovisioned bucket for %s', ref.instance.name)
+            try:
+                ref.instance.provision_s3()
+            except ClientError as e:
+                LOG.error(
+                    'Failled to reprovision bucket for "%s" due to: %s',
```

Comment:
Absolute nittiest of nits: "Failed" should only have one "l".

Reply:
🤦‍♂️